docs/source/quickstart/index.rst

```diff
@@ -184,14 +183,14 @@ It is super easy, just run the follow command::
 
     $ pip3 install --user avocado-framework
 
-.. note:: For more methods, please visit the Install Guide section on our
-          oficial documentation.
+.. note:: For more methods, please visit the Installing section on
+          Avocado User's Guide.
 
 Documentation
 =============
 
-Please see Contents for full documentation, including installation methods,
-tutorials and API.
+Please see the `latest documentation <https://avocado-framework.readthedocs.io/en/latest/index.html>`_
+for full documentation, including installation methods, tutorials and API.
```

Comment:
The only "problem" that I can see here, is that we use this `quickstart/index.rst` on both: README and readthedocs index page.

So if I'm visiting the readthedocs page and click on this link I will be redirected to the same page, right?

Reply:
One other side effect, which I don't know if it's intended, is that that's a difference between the "latest" version (tracks master) and the latest *released* version (which I updated after each release).

Reply:
@beraldoleal , @clebergnu Thank you! I agree. There seem to be three scenarios:

1. User is on github (README, or docs/source/quickstart/index.rst)
2. User is on latest documentation (in development)
3. User is on some release's documentation (who doesn't want to click to another version's documentation)

I think we can serve all by

a. linking to both latest and last release's documentation
b. adding "or browse this site for more content."

quickstart continues to serve both as README and Welcome page.

I updated the PR accordingly. Please, let me know what you think.